Units/parser-vala.r/class.vala.d/input.vala

```diff
@@ -12,12 +12,12 @@ void main(string[] args) {
     print("Hello %s, you're %d years old\n", p.name, p.age);
 }
 
-class Address {
+public class Address {
```

Comment:
FYI, you can store "public"  to "access:" field.

```
[yamato@control ctags-github]$ cat /tmp/foo.cc
cat /tmp/foo.cc
class A {
public:
  int x;
};
[yamato@control ctags-github]$ ./ctags -o - --fields=+'{access}' /tmp/foo.cc
./ctags -o - --fields=+'{access}' /tmp/foo.cc
A	/tmp/foo.cc	/^class A {$/;"	c	language:C++	file:
x	/tmp/foo.cc	/^  int x;$/;"	m	language:C++	class:A	typeref:typename:int	file:	access:public
```

Reply:
Thanks for the hint, I did wonder about this.